src/modelio.cs

```diff
@@ -1638,13 +1638,33 @@ interface MDLTexture : MDLNamed
 		[Export ("init")]
 		IntPtr Constructor ();
 
+#if !XAMCORE_4_0
 		[Static]
-		[Export ("textureNamed:")]
+		[Obsolete ("Use 'FromName' instead.")]
+		[Wrap ("FromName (name)")]
 		MDLTexture FromBundle (string name);
+#endif
 
 		[Static]
-		[Export ("textureNamed:bundle:")]
+		[Export ("textureNamed:")]
+		MDLTexture FromName (string name);
+
+#if !XAMCORE_4_0
+		[Static]
+		[Obsolete ("Use 'FromName' instead.")]
+		[Wrap ("FromName (name, bundleOrNil)")]
 		MDLTexture FromBundle (string name, [NullAllowed] NSBundle bundleOrNil);
+#endif
+
+		[Static]
+		[Export ("textureNamed:bundle:")]
+		MDLTexture FromName (string name, [NullAllowed] NSBundle bundleOrNil);
```

Comment:
While I agree that `FromName` sounds more logical than `FromBundle`, neither has a verb.

If we're renaming, my suggestion would be `CreateTexture` or `CreateNamedTexture` (which is also more similar to the `CreateTextureCube` overloads further below).

Reply:
I agree, thanks for the suggestion, didn't scroll far enough (;